corehq/apps/export/templates/export/export_list.html

```diff
@@ -0,0 +1,467 @@
+                                </span>
+                                <span ng-if="formLoadError !== 'default'">
+                                    {% angularjs %}{{ formLoadError }}{% endangularjs %}
+                                </span>
+                            </div>
+                            <p>
+                                {% blocktrans %}
+                                    If this problem persists, please <a href="#reportIssueModoal"
+                                        data-toggle="modal">Report an Issue</a> and include this error
+                                        message in the description.
+                                {% endblocktrans %}
+                            </p>
+                            <p>
+                                {% blocktrans %}
+                                    We apologize for the inconvenience, and thank you for your patience. Someone
+                                    on our development team is quickly working to resolve this issue.
+                                {% endblocktrans %}
+                            </p>
+                        </div>
+                        <div class="modal-body"
+                             ng-show="!!showNoAppsError">
+                            <p class="lead">
+                            {% blocktrans %}
+                                It seems as though you haven't submitted any data to CommCare HQ.
+                                Please deploy an Application and submit data before creating any exports.
+                            {% endblocktrans %}
+                            </p>
+                            <a href="https://confluence.dimagi.com/display/commcarepublic/Application+Building"
+                               target="_blank">
+                                <i class="fa fa-info-circle"></i>
+                                {% trans 'How to Create and Deploy a CommCare Application.' %}
+                            </a>
+                         </div>
+                        <div class="modal-footer">
+                            <button type="button"
+                                    class="btn btn-default"
+                                    data-dismiss="modal"
+                                    ng-disabled="!!isSubmittingForm">{% trans 'Cancel' %}</button>
+                            <button type="submit"
+                                    class="btn btn-success"
+                                    ng-show="!showNoAppsError && !formLoadError"
+                                    ng-disabled="!!exportOptionsForm.$invalid || !!isSubmittingForm">
+                                {% trans "Create Export" %}
+                            </button>
+                        </div>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+{% endblock %}
```

Comment:
these two HTML files (`export_list`, `download_export`) are monsters. it may help to break out the modals into separate files